mesonbuild/backend/vs2010backend.py

```diff
@@ -884,6 +881,9 @@ def gen_vcxproj(self, target, ofname, guid):
         # vcxproj file (similar to buildtype compiler args) instead of in
         # AdditionalOptions?
         extra_link_args += compiler.get_buildtype_linker_args(self.buildtype)
+        # Generate Debug info
+        if self.buildtype == 'debug':
```

Comment:
Please change this to `self.buildtype.startswith('debug')` and we're good to go.